### ISSUE TITLE
Octave.app Announcements

### ISSUE THREAD

**Issue by apjanke** [octave-app/octave-app#1]
Subscribe to this issue to be notified of updates to Octave.app!

Major Octave.app news is posted as comments on this issue. This includes announcements of new releases, and major organizational changes. This is a low-traffic issue and I'll try to avoid spamming you. It is locked so only the Octave.app maintainers can post to it.

See also [#289 - Octave.app Dev News](https://github.com/octave-app/octave-app/issues/289) for developer-oriented announcements like beta releases and infrastructure updates.

This is a pinned issue which will never be closed. Its purpose is to act like a mailing list for Octave.app news, without me having to set up an actual mailing list or blog.

**Comment by apjanke** [comment]
Octave.app 4.4.0 beta3 is out and ready for testing!

https://github.com/octave-app/octave-app/releases/tag/v4.4.0-beta3

Please address any issues to https://github.com/octave-app/octave-app-bundler/issues.

**Comment by apjanke** [comment]
Octave.app 4.4.0 beta5 is out and ready for testing!

https://github.com/octave-app/octave-app/releases/tag/v4.4.0-beta5

Please address any issues to https://github.com/octave-app/octave-app-bundler/issues.

**Comment by apjanke** [comment]
Octave.app 4.4.0 Release Candidate 2 is out and ready for testing!

https://github.com/octave-app/octave-app/releases/download/v4.4.0-rc2/Octave-4.4.0-RC2.dmg

Please address any issues to https://github.com/octave-app/octave-app-bundler/issues.

**Comment by apjanke** [comment]
Octave.app 4.4.0 is out!

https://github.com/octave-app/octave-app/releases/download/v4.4.0/Octave-4.4.0.dmg

Please address any issues to https://github.com/octave-app/octave-app-bundler/issues.

**Comment by apjanke** [comment]
Octave.app 4.4.1 beta2 is out.

https://github.com/octave-app/octave-app/releases/download/v4.4.1-beta2/Octave-4.4.1-beta2.dmg

Please address any issues to https://github.com/octave-app/octave-app/issues.

**Comment by apjanke** [comment]
Octave.app 4.4.1 beta4 is out.

https://github.com/octave-app/octave-app/releases/tag/v4.4.1-beta4

**Comment by apjanke** [comment]
Octave.app 5.1.0 beta1 is out.

https://github.com/octave-app/octave-app/releases/tag/v5.1.0-beta1

**Comment by apjanke** [comment]
Octave.app 5.1.0 beta 3 is out.

https://github.com/octave-app/octave-app/releases/tag/v5.1.0-beta3

This one is built against Qt 5.11 instead of Qt 5.12, and may fix the stability/freezing issues we've had with 5.1.0 so far.

**Comment by apjanke** [comment]
Octave 4.4.1-u1 beta 2 is out.

https://github.com/octave-app/octave-app/releases/tag/v4.4.1-u1-beta2

This one includes a bundled JDK so it no longer requires the user to install a JDK separately. Also a couple cosmetic improvements.

**Comment by apjanke** [comment]
Octave.app 5.2.0 beta 1 is out.

https://github.com/octave-app/octave-app/releases/tag/v5.2.0-beta1

This is the first beta for Octave 5.2.0. It's built against the new Qt 5.14.1, which may help with the stability/freeziness issues we've been struggling with. Works okay for me so far.

**Comment by apjanke** [comment]
Octave.app 8.4.0 beta 1 is out. Now with native Apple Silicon builds!

https://github.com/octave-app/octave-app/releases/tag/v8.4.0_beta1

There are _special installation instructions_ you must follow for this build, due to security stuff in newer macOS releases. Please see the installation instructions on the release page. If you don't follow them, the installed app won't launch. Sorry for the inconvenience.

**Comment by apjanke** [comment]
Octave.app 8.4.0 Release Candidate 1 is out. If no significant issues are found, this exact build will become the GM 8.4.0 release.

https://github.com/octave-app/octave-app/releases/tag/v8.4.0_RC1

There are _special installation instructions_ you must follow for this build, due to security stuff in newer macOS releases. Please see the installation instructions on the release page. If you don't follow them, the installed app won't launch. Sorry for the inconvenience.

**Comment by apjanke** [comment]
The first Octave.app 9.x prerelease is out: Octave.app 9.1.0 beta1.

https://github.com/octave-app/octave-app/releases/tag/v9.1.0_beta1

This is our first 9.x build, and we're still working on getting 8.4.0 out, so it may be a while before this one goes GM. We're also trying to migrate it from Qt 5 to Qt 6 for 9.x, which isn't working yet, but there's some hope.

**Comment by apjanke** [comment]
Octave.app 8.4.0 is out!

https://github.com/octave-app/octave-app/releases/tag/v8.4.0

NOTE: There are _special installation instructions_ you must follow for this build, due to security stuff in newer macOS releases. Please see the installation instructions on the release page. If you don't follow them, the installed app won't launch. Sorry for the inconvenience.

**Comment by apjanke** [comment]
I created a separate developer-oriented [#289 "Octave.app Dev News"](https://github.com/octave-app/octave-app/issues/289) issue, to split out developer-oriented news from end-user-facing announcements. Future announcements of betas and other prereleases will be posted there instead. Only "GM" ("golden master" or final) releases will continue to be announced on this here issue. This should cut down on noise for end users who only care about "real" releases, and avoid having end users accidentally install preview releases that aren't yet ready for prime time and may be buggy.

If you're a developer working on Octave.app or packages that want to support it, or you're a power user who wants to help test new Octave.app releases while they're under development, please subscribe to #289 to be notified of maintainer-oriented prereleases.

**Comment by apjanke** [comment]
Octave.app 9.1 and 9.2 are out!

* https://github.com/octave-app/octave-app/releases/tag/v9.1
* https://github.com/octave-app/octave-app/releases/tag/v9.2

These are the first Octave.app releases of the 9.x series. Version 9.2 is recommended.

## Special Installation Instructions - REQUIRED!

TL;DR: Use `xattr -c` to un-quarantine the DMG before opening it.

After downloading the DMG, before opening it (double-clicking it) to do the installation, you must first "un-quarantine" it. Open Terminal and run `xattr -c ~/Downloads/Octave-*.dmg`. Then you can double-click the DMG file and proceed to install it as usual. If you don't do this extra `xattr -c` step, it will probably just give you an "app is damaged" error instead of launching when you try to run it from /Applications. Sorry; working on fixing this, but not super hopeful. See [#244 – App "damaged" for GitHub downloads](https://github.com/octave-app/octave-app/issues/244).
